build/dockerfiles/Dockerfile

```diff
@@ -11,16 +11,22 @@
 #
 
 # Builder: check meta.yamls and create index.json
-FROM docker.io/alpine:3.11.5 AS builder
-RUN apk add --no-cache py3-pip jq bash wget skopeo && pip3 install yq jsonschema
+FROM docker.io/node:12.19.0-alpine3.12 AS builder
+RUN apk add --no-cache py3-pip jq bash git wget skopeo && pip3 install yq jsonschema
```

Comment:
Is the `builder` published somewhere ? if not, could we publish it so it would be available in a devfile.yaml ? (for dogfooding, generating the plugins and start a plugin registry in Che)

Reply:
no but this is one of my goal as this repository had not be designed to use dogfooding and do almost everything in a docker containr